### ISSUE TITLE
All disks are added as attributes

### ISSUE THREAD

**Issue by PW999** [bkbilly/lnxlink#68]
### What type of installation are you running?

Desktop

### Which Linux OS are you using?

Manjaro 6.1

### Which version of LNXLink has the issue?

2023.8.0

### Describe the problem

As you can see in the below screenshot, my disk partitions are discovered successfully
![image](https://github.com/bkbilly/lnxlink/assets/8831154/dfa0016a-0f2f-426e-aff3-9c46cfa5d496)

When I check the attributes of one of those sensors, I see all my partitions again
![image](https://github.com/bkbilly/lnxlink/assets/8831154/ba6396d6-3986-4864-aff1-da1241d5a06c)

I'm not 100% sure if those are exposed as state attributes, but according to the docs, if those change frequently it may cause a significant database size increase: https://developers.home-assistant.io/docs/core/entity/

### Configuration

```yaml
mqtt:
  prefix: lnxlink
  clientId: PWManjaro
  statsPrefix: monitor/stats
  server: xxx
  port: 1883
  auth:
    user: xxx
    pass: xxx
  discovery:
    enabled: true
  lwt:
    enabled: true
    qos: 1
    retain: true
    connectMsg: 'ON'
    disconnectMsg: 'OFF'
update_interval: 5
modules:
  - suspend
  - disk_usage
  - idle
  - memory
  - gpu
  - screen_onoff
  - network
  - notify
  - cpu
  - media
  - shutdown
  - restart
  - xdg_open
```

### Logs

```bash
aug 14 09:53:48 PWManjaro systemd[812]: Started LNXlink.
aug 14 09:53:48 PWManjaro lnxlink[8851]: INFO:lnxlink:LNXlink 2023.8.0 started: 3.11.3
aug 14 09:53:48 PWManjaro lnxlink[8851]: INFO:lnxlink:Loaded addons: suspend, disk_usage, idle, memory, gpu, screen_onoff, network, notify, cpu, media, shutdown, restart, xdg_open
aug 14 09:53:48 PWManjaro lnxlink[8851]: INFO:lnxlink:Connected to MQTT with code 0
aug 14 09:55:20 PWManjaro lnxlink[8851]: INFO:lnxlink:Message received screen_onoff/Screen_OnOff/: b'OFF'
```

**Comment by bkbilly** [comment]
Currently it is not supported to disable the attributes, but I will look into it.
Thanks for letting me know about this.

**Comment by bkbilly** [comment]
I've made some changes on this, but I didn't get the chance to test them out.
Now every entity should have it's own attributes.

**Comment by PW999** [comment]
I had to remove LNXLink from the MQTT integration and re-add it before the changes showed up, but it's working like a charm now
![image](https://github.com/bkbilly/lnxlink/assets/8831154/2d59e438-2b7f-48e0-8c9a-2056f430de37)

**Comment by bkbilly** [comment]
Fixed on LNXlink version `2023.9.0`.
There is an issue that if a disk is disconnected, it will show errors on home assistant.
I will have to look how to fix it in the future, but for now it should work.
